scripts/post-install/post-install.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const Logger = require('../utils/logger');
+
+fs.readFile(`${process.env.INIT_CWD}/package.json`, (error, file) => {
+  if (error) {
+    throw Logger.error(error);
```

Comment:
Should we silently return null if theres an error?

Reply:
We can, but we might start having issues we never know about

Reply:
won't it come from reading the file though? I'd hate to error on people's installs

Reply:
We can grab a few more opinions on this before merging. I don't mind removing the throw, but the trade off is that we will never be alerted this isn't working correctly.